### PR TITLE
Add `…--weekend` class to Weekdays tiles

### PR DESCRIPTION
**Pull request by fffed**
Add  '--weekend' class to `weekday` so we could style it differently like on a month view for dates.

### CODE REVIEW

**Comment by fffed** [comment]
Feel free to reject it ))

**Comment by Chaoste** [comment]
Code LGTM, would also love to have that.
Right now we're using `:nth-child(6+n)` which feels hacky 😅

File: src/MonthView/Weekdays.jsx [hunk 1]
```diff
@@ -34,12 +35,13 @@ export default function Weekdays(props) {
       year, monthIndex, weekday - getDayOfWeek(beginOfMonth, calendarType),
     );
 
+    const weekdayClassName = `${className}__weekday`;
```
Comment (wojtekmaj) [inline src/MonthView/Weekdays.jsx:38]: Move to line 17; there's no need to re-create this variable on every render, let alone in a loop!

File: src/MonthView/Weekdays.jsx [hunk 2]
```diff
@@ -34,12 +35,13 @@ export default function Weekdays(props) {
...
     const abbr = formatWeekday(locale, weekdayDate);
 
     weekdays.push(
       <div
         key={weekday}
-        className={`${className}__weekday`}
+        className={mergeClassNames(weekdayClassName, isWeekend(weekdayDate) && `${weekdayClassName}--weekend`)}
```
Comment (wojtekmaj) [inline src/MonthView/Weekdays.jsx:44]: `calendarType` is required in `isWeekend`.

```suggestion
        className={mergeClassNames(
          weekdayClassName, 
          isWeekend(weekdayDate, calendarType) && `${weekdayClassName}--weekend`
        )}
```

**Review by wojtekmaj — CHANGES_REQUESTED** [review]
Neat - although needs a bit more work as weekends are differently defined for Arabic and Hebrew calendars :)

**Review by wojtekmaj — APPROVED** [review]
Great! Thanks!

**Pull request merged**